cppser/app.cpp

```diff
@@ -55,10 +55,22 @@ namespace Spirit {
             }
             return true;
         };
+        auto check_bool = [&config](const char* entry) {

```

Comment:
Now we have three similar lambdas doing essentially the same job.

Reply:
Maybe we can leave this to the next project. Currently I am not of ideas apart from member function pointers.